schema/commonmeta/commonmeta.schema.json

```diff
@@ -0,0 +1,541 @@
+            }
+        },
+        "geo_locations": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "geoLocationPlace": {
+                        "type": "string"
+                    },
+                    "geoLocationPoint": {
+                        "$ref": "#/definitions/geoLocationPoint"
+                    },
+                    "geoLocationBox": {
+                        "type": "object",
+                        "properties": {
+                            "westBoundLongitude": {
+                                "$ref": "#/definitions/longitude"
+                            },
+                            "eastBoundLongitude": {
+                                "$ref": "#/definitions/longitude"
+                            },
+                            "southBoundLatitude": {
+                                "$ref": "#/definitions/latitude"
+                            },
+                            "northBoundLatitude": {
+                                "$ref": "#/definitions/latitude"
+                            }
+                        },
+                        "required": [
+                            "westBoundLongitude",
+                            "eastBoundLongitude",
+                            "southBoundLatitude",
+                            "northBoundLatitude"
+                        ]
+                    },
+                    "geoLocationPolygons": {
+                        "type": "array",
+                        "items": {
+                            "type": "object",
+                            "properties": {
+                                "polygonPoints": {
+                                    "type": "array",
+                                    "items": {
+                                        "$ref": "#/definitions/geoLocationPoint"
+                                    },
+                                    "minItems": 4
+                                },
+                                "inPolygonPoint": {
+                                    "$ref": "#/definitions/geoLocationPoint"
+                                }
+                            },
+                            "required": [
+                                "polygonPoints"
+                            ]
+                        },
+                        "uniqueItems": true
+                    }
+                }
+            },
+            "uniqueItems": true
+        },
+        "provider": {
+            "description": "The provider of the resource. This can be a DOI registration agency or a repository.",
+            "type": "string",
+            "enum": [
+                "Crossref",
+                "DataCite",
+                "GitHub",
+                "JaLC",
+                "KISTI",
+                "mEDRA",
+                "OP"
+            ]
+        },
+        "alternate_identifiers": {
```

Comment:
I've merged this into the `related_identifiers` field

Reply:
Using an `isIdenticalTo` relation type?

Reply:
Hm, that would simplify the schema. I will think more about and adjust the commonmeta schema.

Reply:
My general thinking is to replace related_identifiers with several more specific sub-properties. References is one of them, for datasets versions and hasPart/isPartOf are super important.

Reply:
Related_identifiers work well as a general concept, but there are too many relation_types. And for converting to/from other metadata formats they be tricky, requiring a lot of extra logic (my experience with bolognese and earlier versions of commonmeta-py.